### PR TITLE
Jormungandr: forward uri when not findable

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

**Review by pbougue — COMMENTED** [review]
Not sure to understand a use case for that PR. Can it affect other endpoints?

**Comment by xlqian** [comment]
@TeXitoi `new_default` and `experimental` return exactly the same response :tada:

**Comment by xlqian** [comment]
@pbougue you can try this request: http://canaltp.github.io/navitia-playground/play.html?request=http%3A%2F%2Fnavitia2-ws.ctp.dev.canaltp.fr%2Fv1%2Fcoverage%2Ffr-auv%2Fjourneys%3Ffirst_section_mode%255B%255D%3Dcar%26from%3D2.9201546%253B45.8603889%26count%3D1%26to%3D3.0029581%253B45.8646066%26max_duration%3D0%26last_section_mode%255B%255D%3Dcar%26_override_scenario%3Dexperimental%26

in this request, `from` is a place which has no label/name/id, so this line https://github.com/CanalTP/navitia/pull/1885/files#diff-05cbd102661a0da9e75756727f9715bfL45 crashed like a comet....

File: source/jormungandr/jormungandr/georef.py [hunk 1]
```diff
@@ -44,11 +44,13 @@ def place(self, place):
         response = self.instance.send_and_receive(req)
         if response.places:
             return response.places[0]
-        # In some cases, the given "from" is not always findable by kraken
-        # we just forward the given uri and return it with a pt object
-        p = type_pb2.PtObject()
-        p.uri = place
-        return p
+        if "coord:" in place:
```
Comment (TeXitoi) [inline source/jormungandr/jormungandr/georef.py]: `place.startswith("coord:")`?

**Review by TeXitoi — CHANGES_REQUESTED** [review]
Tests please.

File: source/jormungandr/tests/journey_common_tests.py [hunk 2]
```diff
@@ -559,6 +559,16 @@ def test_journey_stop_area_to_stop_point(self):
         assert j['sections'][0]['to']['id'] == 'stop_point:stopB'
         assert 'walking' in j['tags']
 
+    def test_journey_from_non_valid_stop_area(self):
+        """
+        When the departure is stop_area:A and the destination is stop_point:B belonging to stop_area:B
```
Comment (TeXitoi) [inline source/jormungandr/tests/journey_common_tests.py]: unsyncronised comment

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**